linera-client/src/client_options.rs

```diff
@@ -611,6 +611,11 @@ pub enum ClientCommand {
         /// provided fixed BPS rate.
         #[arg(long)]
         bps: Option<usize>,
+
+        /// If provided, will not close the chains after the benchmark is finished. This is useful
+        /// when running with many chains, as closing them all might take a while.
+        #[arg(long)]
+        leak_chains: bool,
```

Comment:
Maybe there is no need to introduce the concept of "leaking chains" and we should just write `--do-not-close-chains` / `do_not_close_chains`